tests/prague/eip6110_deposits/helpers.py

```diff
@@ -5,9 +5,11 @@
 from hashlib import sha256 as sha256_hashlib
 from typing import Callable, ClassVar, List
 
-from ethereum_test_tools import EOA, Address, Alloc, Bytecode, Hash, Transaction
+from ethereum_test_tools import EOA, Address, Alloc, Bytecode
 from ethereum_test_tools import DepositRequest as DepositRequestBase
+from ethereum_test_tools import Hash
 from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools import Transaction
```

Comment:
```suggestion
from ethereum_test_tools import EOA, Address, Alloc, Bytecode, Hash, Transaction
from ethereum_test_tools import DepositRequest as DepositRequestBase
from ethereum_test_tools import Opcodes as Op
```
Just to appease the linter.